data/presets/presets/highway/secondary_link.json

```diff
@@ -3,7 +3,7 @@
     "fields": [
         "name",
         "oneway",
-        "maxspeed",
+        "maxspeed/advisory",
```

Comment:
maxspeed=* is a valid tag on higway=secondary_link in some countries and should not be removed. `highway=secondary_link` in settlements where the normal urban speed limit (e.g. 50 kph in many European countries) applies should get `maxspeed=50` or `maxspeed=<country_code>:urban`.